src/LanguageServerProtocol.fs

```diff
@@ -3000,7 +3000,7 @@ module Server =
 module Client =
     open System
     open System.IO
-    open LanguageServerProtocol.Logging
+    open FsLibLog
```

Comment:
We should rename this namespace back to LanguageServerProtocol.Logging here and in the FsLibLog.fs file, to maintain the API surface.